gateway-service-knoxtoken/src/test/java/org/apache/knox/gateway/service/knoxtoken/TokenServiceResourceTest.java

```diff
@@ -778,6 +785,32 @@ public void testTokenRevocation_Enabled_WithRenewersWithValidSubject() throws Ex
     validateSuccessfulRevocationResponse(renewalResponse);
   }
 
+  @Test
+  public void testGetPublicKey() throws Exception {
+    TokenResource tr = createTokenResourceForPublicKeyTests(false);
+
+    Response response = tr.publicKey();
+    assertEquals("Unexpected response code", Status.OK.getStatusCode(), response.getStatus());
+    String retString = response.getEntity().toString();
+    Map<String, String> responseMap = parseJSONResponse(retString);
+    assertNotNull("PublicKey is null", responseMap.get(TokenResource.PUBLIC_KEY));
+    assertNotNull("Algorithm is null", responseMap.get(TokenResource.ALGORITHM));
+    assertEquals("Mismatched algorithm", responseMap.get(TokenResource.ALGORITHM),
```

Comment:
Nitpick: Generally, the expected value is first in assertEquals(expected, actual)

Reply:
Won't count that as a nitpick. Expected should go first per the API. Fixed.